chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/confirm-dialog/README.md

# @vaadin/confirm-dialog

> A modal dialog web component used to confirm user actions.

Reviewer: ```suggestion
A modal dialog web component for confirming user actions.
```

Author: Done.
